### PR TITLE
Update to use recommended license comment style

### PR DESCRIPTION
**Pull request by rictic**
As I was instructed by the google third party code folks.

### CODE REVIEW

File: paper-checkbox.css [hunk 1]
```diff
@@ -1,11 +1,12 @@
-/*
-Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
-This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-Code distributed by Google as part of the polymer project is also
-subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
-*/
+/**
+ * @license
```
Comment (notwaldorf) [inline paper-checkbox.css]: I don't think we indent anywhere else.

**Comment by notwaldorf** [comment]
LGTM % indentation

**Comment by rictic** [comment]
How's this?

**Comment by notwaldorf** [comment]
Nailed it :shipit:

**Pull request merged**